### PR TITLE
Fixed issue #10080: Bad look of multiple text with "Input box width : 2"

### PR DESCRIPTION
**Pull request by Shnoulle**
Dev: the 10080 issue does what "column number" of single radio do.

### CODE REVIEW

**Comment by Shnoulle** [comment]
Screen with tyhis fix.

test_mult_width_-_2015-12-10_19 03 40

File: application/helpers/qanda_helper.php [hunk 1]
```diff
@@ -2991,9 +2989,9 @@ function do_multipleshorttext($ia)
                 list($htmltbody2, $hiddenfield)=return_array_filter_strings($ia, $aQuestionAttributes, $thissurvey, $ansrow, $myfname, '', $myfname, "li","question-item answer-item text-item".$extraclass);
 
                 $answer_main .= "\t$htmltbody2\n"
-                . "<label for=\"answer$myfname\">{$ansrow['question']}</label>\n"
+                . "<label for=\"answer$myfname\" class=\"col-sm-6\">{$ansrow['question']}</label>\n"
                 . "\t<span>\n".$prefix."\n".'
-                <textarea class="form-control  textarea '.$kpclass.'" name="'.$myfname.'" id="answer'.$myfname.'"
+                <textarea class="form-control  textarea col-sm-6'.$kpclass.'" name="'.$myfname.'" id="answer'.$myfname.'"
```
Comment (Shnoulle) [inline application/helpers/qanda_helper.php]: Think the form-control col-sm-6 MUST be to the span ( see bootstrap example)

**Comment by kerstin** [comment]
Was this fixed? Issues for the repository aren't visible to third parties... and there's no information about this having got merged either.

Thx.

**Comment by Shnoulle** [comment]
@kerstin : it's not fixed ..... 2.56 have same issue . In develop : adding a new attribute (input_size), but you must control old settings when update from 2.6

**Pull request closed without merging**